src/main/java/com/training/tinyurl/service/TinyUrlServiceImpl.java

```diff
@@ -34,4 +37,16 @@ public void createNewUser(RegistrationReqDto request) throws MongoApiException {
         log.error("User with same email already exists");
         throw new MongoApiException(HttpStatus.CONFLICT,"User with same email already exists");
     }
+
+    @Override
+    public AppUserDetails getLoggedInUserDetails(){
```

Comment:
This method is actually not required.
@AuthenticationPrincipal can be used to get authenticated user.